src/renderer/msg_filter.rs

```diff
@@ -0,0 +1,74 @@
+//! This module contains logic for filtering windows messages.
+
+use bitflags::bitflags;
+use windows::Win32::UI::WindowsAndMessaging::*;
+
+bitflags! {
+    /// Flags for filtering window messages.
```

Comment:
Could you please add a long-form explanation to this, with examples, so users can get a better idea at a glance of what this data structure is for and how it should be used?